moveit_experimental/moveit_jog_arm/config/ur_simulated_config.yaml

```diff
@@ -54,4 +54,3 @@ command_out_topic: joint_group_position_controller/command # Publish outgoing co
 check_collisions: true # Check collisions?
 collision_check_rate: 5 # [Hz] Collision-checking can easily bog down a CPU if done too often.
```

Comment:
This rate should probably be increased. Maybe it would be wise to add a warning to the thread initialization in case this rate is very low.

Reply:
OK, will fix